### PR TITLE
EWS - changes in location filter

### PR DESCRIPTION
**Pull request by lwyszomi**
@czue 
http://manage.dimagi.com/default.asp?169919#965941

### CODE REVIEW

File: custom/ewsghana/filters.py [hunk 1]
```diff
@@ -185,4 +191,4 @@ def filter_context(self):
             selected_type=self.selected('type') if self.selected('type') else 2,
             selected_first=self.selected('first') if self.selected('first') else self.default_week,
             selected_second=self.selected('second') if self.selected('second') else ''
-        )
+        )
```
Comment (dimagimon) [inline custom/ewsghana/filters.py]: W292 no newline at end of file

File: custom/ewsghana/templates/ewsghana/partials/location_async.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{% load hq_shared_tags %}
```
Comment (czue) [inline custom/ewsghana/templates/ewsghana/partials/location_async.html]: can you clarify what the difference is between this and https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/templates/reports/filters/location_async.html is? any way for this one to extend that one?

**Comment by czue** [comment]
(also no longer mergeable)

File: custom/ewsghana/templates/ewsghana/partials/location_async.html (hunk 2 above)
Comment (lwyszomi) [inline custom/ewsghana/templates/ewsghana/partials/location_async.html]: @czue I removed ```disable: !can_edit_children()```

**Comment by lwyszomi** [comment]
@czue updated

File: custom/ewsghana/templates/ewsghana/partials/location_async.html (hunk 2 above)
Comment (czue) [inline custom/ewsghana/templates/ewsghana/partials/location_async.html]: i see. can you just add a section to the base template and then override it here?

Comment (lwyszomi) [inline custom/ewsghana/templates/ewsghana/partials/location_async.html]: @czue  sure

**Comment by lwyszomi** [comment]
@czue updated

**Comment by czue** [comment]
thanks. :+1: to merge when tests pass

**Comment by czue** [comment]
arg, not mergeable again

**Comment by lwyszomi** [comment]
@czue updated

**Pull request merged**